<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Benutzerdaten</class>
 <widget class="QDialog" name="Benutzerdaten">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Benutzerdaten Ã¤ndern</string>
  </property>
  <widget class="QPushButton" name="btn_NuSpeichern">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_NuAbbrechen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vorname:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_benutzerDataVname">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_benutzerDataName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nachname:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Geburtsdatum:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="txt_benutzerDataGeb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_benutzerDataVname</tabstop>
  <tabstop>txt_benutzerDataName</tabstop>
  <tabstop>txt_benutzerDataGeb</tabstop>
  <tabstop>btn_NuSpeichern</tabstop>
  <tabstop>btn_NuAbbrechen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
